src/main/java/com/example/todolist/Domain/Entity/Task.java

```diff
@@ -0,0 +1,50 @@
+package com.example.todolist.Domain.Entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Inheritance;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+@Entity
+@Inheritance
+public class Task extends BaseEntity{
+    @NotNull
+    @NotBlank
```

Comment:
Create a rule called TitleRule and use it instead, this is not the same case as Password